frontend/src/pages/article/components/article-form/article-form.tsx

```diff
@@ -50,19 +67,44 @@ const ArticleForm: React.FC = () => {
     [dispatch, generatedPrompt],
   );
 
+  const handleArticleUpdate = useCallback(
+    (payload: ArticleRequestDto): void => {
+      if (!articleForUpdate) {
+        return;
+      }
+      const updatePayload = {
+        articleId: articleForUpdate.id,
+        articleForUpdate: { text: payload.text, title: payload.title },
+        navigate,
+      };
+      void dispatch(articlesActions.updateArticle(updatePayload));
```

Comment:
```suggestion
      void dispatch(articlesActions.updateArticle(updatePayload))
        .unwrap()
        .then(() => {
          navigate
        });
```

Reply:
isn't unwrap will throw errors as well ?

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/44391531/9d3212ba-120c-4e78-b851-8786e17bf5de)

This is unwrap. Should I leave it like that ? Or add some empty catch ?